inc/dashboard/provider.class.php

```diff
@@ -454,21 +454,21 @@ public static function nbTicketsByAgreementStatusAndTechnician(
          ],
          'FROM'   => $table,
          'INNER JOIN' => [
-            "$ticketUserTable as gu" => [
+            "$ticketUserTable as ul" => [
                'FKEY' => [
-                  'gu' => 'tickets_id',
+                  'ul' => 'tickets_id',
                   $table => 'id',
                   [
                      'AND' => [
-                        "gu.type" => Ticket_User::ASSIGN
+                        "ul.type" => Ticket_User::ASSIGN
                      ]
                   ]
                ],
             ],
             $userTable => [
                'FKEY' => [
                   $userTable => 'id',
-                  'gu' => 'users_id',
+                  'ul' => 'users_id',
```

Comment:
related to this code : https://github.com/glpi-project/glpi/blob/master/inc/dashboard/provider.class.php#L1842